src/components/Register.jsx

```diff
@@ -21,109 +22,171 @@ const Register = () => {
       return;
     }
     const cardElement = elements.getElement(CardElement);
-    stripe.createToken(cardElement).then((result) => {
-      Authentication.subscribe(event, result, setLoading);
-    });
+    stripe
+      .createPaymentMethod({ type: 'card', card: cardElement })
+      .then((stripeDetails) => {
+        Authentication.subscribe(
+          event,
+          stripeDetails,
+          setLoading,
+          subscriptionPlan
+        );
+      });
   };
 
+  const subscriptionPlans = [
+    {
+      id: 'monthly_subscription',
+      title: 'MONTHLY SUBSCRIPTION',
+      price: '130 SEK / month',
+      info: 'Gain access to premium content and more...',
+    },
+    {
+      id: 'half_year_subscription',
+      title: 'HALF YEAR SUBSCRIPTION',
+      price: '110 SEK / month',
+      info: 'Save 15% by purchasing 6 month subscription',
+    },
+    {
+      id: 'yearly_subscription',
+      title: 'YEARLY SUBSCRIPTION',
+      price: '100 SEK / month',
+      info: 'Save another 10% by purchasing 12 month subscription',
+    },
+  ];
+
+  const subscriptionPlanCardList = subscriptionPlans.map((plan, index) => {
+    return (
+      <div
+        onClick={() => {
+          setSubscriptionPlan(plan.id);
+        }}
+        className={
+          index === 1
+            ? 'box-shadow subscription-special'
+            : 'box-shadow subscription-plan'
+        }
+        data-cy='subscription-plan'
+        key={index}>
+        <div className='subscription-header'>
+          <h4>{plan.title}</h4>
+        </div>
+        <div className='subscription-info'>
+          <h2>{plan.price}</h2>
+          <p>{plan.info}</p>
+        </div>
+      </div>
+    );
+  });
+
   return (
     <>
       {subscriber && <AuthenticationMessage time={5000} />}
+      <h1 style={{ color: 'white', textAlign: 'center' }}>
+        {subscriptionPlan ? 'Fill in your details' : 'Choose a subscription:'}
+      </h1>
       <Segment
         placeholder
         textAlign='center'
-        style={{ backgroundColor: '#202325', height: 600 }}>
+        style={{ backgroundColor: '#111518', height: 500 }}>
         {location.state && (
           <h1
             style={{ marginBottom: 50, color: 'white' }}
             data-cy='subscribe-message'>
             You tried to access a premium article, please subscribe
           </h1>
         )}
+        {!subscriptionPlan && (
+          <>
+            <Card.Group centered>{subscriptionPlanCardList}</Card.Group>
```

Comment:
Does Card.Group works if you dont use Cards?

Reply:
Apparently, yeah!